fetch.bs

```diff
@@ -7480,6 +7485,48 @@ are:
  <li><p>Return <var>responseObject</var>.
 </ol>
 
+
+<p>The static
+<dfn method for=Response><code>json(<var>data</var>, <var>init</var>)</code></dfn> method steps
```

Comment:
The implementation here has a lot of duplication with `new Response`. I could rewrite this as follows if that is preferred (not real spec text):

1. Create new header list.
2. Fill header list with init.headers.
3. If header list does not contain `content-type`, set it to `application/json`.
4. Let body be `JSON.stringify(body)`.
5. Set init.headers = header list.
5. Return `new Response(body, init)`

Reply:
Reusing the steps makes sense to me, but note that we cannot invoke global constructors in the way the pseudo-code suggests. It would have to be some kind of shared abstraction.

Reply:
Ok, rewrote this so the shared steps are extracted into a common "initialize a response" algorithm.